services/dask-sidecar/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# DEVELOPMENT recipes for Workflows manager service
+#
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+
+TEMP_DIR := $(shell mktemp -d -t dask-docker-XXX)
+
+PHONY: build-official-dask
+#
+# NOTE:  At this moment, this does not seem to work https://docs.docker.com/engine/reference/commandline/build/
+# export DOCKER_BUILDKIT=0; docker build --tag local/dask:master https://github.com/dask/dask-docker.git#:base
+#
+build-official-dask: # builds official dask container from master branch repo
```

Comment:
Why is this built from the master branch and not from the same stable release?

Reply:
This is here just for the sake of comparison ... for development. It has no impact at all with our integration of dask